src/test/javascript/javascript-import-scanner_test.ts

```diff
@@ -40,6 +40,24 @@ suite('JavaScriptImportScanner', () => {
     assert.equal(features.length, 1);
     assert.equal(features[0].type, 'js-import');
     assert.equal(features[0].url, '/static/javascript/submodule.js');
+    assert.equal(features[0].lazy, false);
+  });
+
+  test('finds dynamic imports', async() => {
+    const file = fs.readFileSync(
+        path.resolve(__dirname, '../static/javascript/dynamic-import.js'),
+        'utf8');
+    const document = parser.parse(
+        file, '/static/javascript/dynamic-import.js' as ResolvedUrl);
+
+    const visit = (visitor: Visitor) =>
+        Promise.resolve(document.visit([visitor]));
+
+    const {features} = await scanner.scan(document, visit);
+    assert.equal(features.length, 1);
+    assert.equal(features[0].type, 'js-import');
+    assert.equal(features[0].url, '/static/javascript/submodule.js');
+    assert.equal(features[0].lazy, true);
```

Comment:
This style of test is legible, but it fails poorly (and I know we do it elsewhere in this file).

Prefer asserting against the interesting subset (or with deepEqual, though for anything with a source range that's not best).

e.g. consider the case where two imports are found. Currently the error message would be `expected 2 to be 1`, but if written something like this:

```typescript
assert.subsetOf(features, [{type: 'js-import', url: '/static/javascript/submodule.js', lazy: true}]);
```

The error would be actionable, you'll see a printout of both imports found, with a nice diff rendering of expected vs actual.

Reply:
Awesome suggestion.  Switched both tests to use `assert.constainSubset()` (`subsetOf` not found?)